### ISSUE TITLE
[UX2.0] Configuration Group state refresh fails if config group was deleted in UI

### ISSUE THREAD

**Issue by tzarski0** [CiscoDevNet/terraform-provider-sdwan#297]
When configuration group was created by terraform and then removed in UI, the state refresh fails with:
```
│ Error: Client Error
│ 
│   with module.sdwan.sdwan_configuration_group.configuration_group["single_device"],
│   on ../terraform-sdwan-nac-sdwan/sdwan_configuration_groups.tf line 1, in resource "sdwan_configuration_group" "configuration_group":
│    1: resource "sdwan_configuration_group" "configuration_group" {
│ 
│ Failed to retrieve object (GET), got error: HTTP Request failed: StatusCode 400, {"error":{"message":"Invalid config
│ group passed","code":"CFGRP0006","details":"Group with id 30208116-ef73-4bc1-9459-4b62ef498f1f of GroupType:
│ config-group not found.","type":"error"}}
```

**Comment by danischm** [comment]
Fix: https://github.com/CiscoDevNet/terraform-provider-sdwan/commit/520f3412ec42bda2b41869f167e5d917ddeef270

**Comment by tzarski0** [comment]
Fixed in 0.4.1.
